AlpsSDK/AlpsManager/AlpsManager.swift

```diff
@@ -13,8 +13,8 @@ import Alps
 open class AlpsManager: MatchMonitorDelegate, ContextManagerDelegate {
 
     let apiKey: String
-    lazy var contextManager = ContextManager(delegate: self)
```

Comment:
Inferred type is better. Less code, same readablity.

Reply:
I added it because I couldn't build without specifying the type. 
What version of Xcode you have ?